### PR TITLE
[11.0] [MIG] maintenance_plan

### PR DESCRIPTION
**Pull request by dalonsod**
Superseeds #11 , fixes cron job issue with demo data

### CODE REVIEW

File: maintenance_plan/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {'name': 'Maintenance Plan',
  'summary': 'Extends preventive maintenance planning',
- 'version': '10.0.1.0.0',
- 'author': 'Odoo Community Association (OCA), Camptocamp SA',
+ 'version': '11.0.1.0.0',
+ 'author': 'Odoo Community Association (OCA), Camptocamp SA, Appcademy SRL',
```
Comment (astirpe) [inline maintenance_plan/__manifest__.py]: I don't think `Appcademy SRL` should be mentioned as "Author" here, as the contribution was a basic porting of the module. The Contributors list was already updated and IMO that would be enough.

**Review by astirpe — CHANGES_REQUESTED** [review]
Tested in Runbot, the issue with the cron and demo data is fixed. Thanks!

File: maintenance_plan/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply maintenance_plan/__manifest__.py]: I agree. Please put it on contributors only

Comment (dalonsod) [reply maintenance_plan/__manifest__.py]: Done!

**Comment by astirpe** [comment]
@pedrobaeza it seems to me that tests are not executed at all in this V11 branch. In there something wrong in the configuration of this repo?

**Comment by pedrobaeza** [comment]
That's because https://github.com/OCA/maintenance/blob/7e6854e8f25654cd4a357e8877905f3c7ed021ce/maintenance_plan/tests/__init__.py is empty and it's not importing test file.

**Comment by dalonsod** [comment]
OK, one test is not passed yet. I'll fix it

**Comment by astirpe** [comment]
Thanks @pedrobaeza 
@dalonsod thanks, you should be mentioned in the contributors list of the README as well

**Comment by dalonsod** [comment]
@astirpe it seems everything's ok. Could you check?

**Review by astirpe — APPROVED** [review]
Tried on Runbot: seems working fine. Thanks!

**Pull request merged**